editor/store/reducer.js

```diff
@@ -668,6 +668,7 @@ export function metaBoxes( state = defaultMetaBoxState, action ) {
 					...state[ location ],
 					isLoaded: false,
 					isActive: action.metaBoxes[ location ],
+					html: getLocationHtml( location ),
```

Comment:
Rather than calling into DOM from the reducer:

- Could the `MetaBoxesArea` component dispatch a "set HTML" action when it mounts?
- Should this at least be modeled as an effect? (Minding that with #3983 we should respect that effects occur after the original action)